CHANGELOG.md

```diff
@@ -1,3 +1,45 @@
+## 10.0.0 (YYYY-MM-DD)
+
+NOTE: This is a unified release note covering all v10.0.0-BETA.X v10.0.0-RC.X releases.
+
+NOTE: Support for syncing with realm.cloud.io and/or Realm Object Server has been replaced with support for syncing with MongoDB Realm Cloud.
+
+NOTE: This version uses the Realm file format to version 20. It is not possible to downgrade to earlier versions than v10.0.0-BETA.7. Non-sync Realms will be upgraded automatically. Synced Realms can only be automatically upgraded if created with Realm JavaScript v10.0.0-BETA.1 and above.
+
+### Breaking Changes
+* [RealmApp] Most API's for interacting with Realm Cloud has changed significantly. All new API's can be found in the `io.realm.mongodb` package. The entry point is though the `App` class from which you can create and login users and otherwise interact with MongoDB Realm. See [the docs](https://docs.mongodb.com/realm/android/) for further details. Synced Realms still use a `SyncConfiguration` that are largely created the same way.
```

Comment:
```suggestion
* [RealmApp] Most APIs for interacting with Realm Cloud have changed significantly. All new APIs can be found in the `io.realm.mongodb` package. The entry point is though the `App` class from which you can create and login users and otherwise interact with MongoDB Realm. See [the docs](https://docs.mongodb.com/realm/android/) for further details. Synced Realms still use a `SyncConfiguration` that are largely created the same way.
```

https://news.ycombinator.com/item?id=3352411